File: executable/python/horizontal-histogram.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('test-runner', '9h29fe3l0x')

import numpy as np
y = np.random.randn(500)

data = Data([
    Histogram(
        y=y
    )
])
plot_url = py.plot(data, filename='horizontal-histogram')

File: executable/julia/horizontal-histogram.jl
using Plotly

using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

y = randn(500)

data = [
  [
    "y" => y, 
    "type" => "histogram"
  ]
]

response = Plotly.plot([data], ["filename" => "horizontal-histogram", "fileopt" => "overwrite"])
plot_url = response["url"]